src/scss/components/_progress-indicator.scss

```
@@ -24,6 +24,8 @@
  */
 .openforms-progress-indicator {
   @include card.card('progress-indicator');
+  overflow: auto;
+  max-height: 100vh;
```

```suggestion
  overflow: auto;
  max-height: 100vh;
```
this breaks the story http://localhost:6006/?path=/story/private-api-progressindicator--default on "Small mobile" viewport, presumably because the screen height is less than 100vh - but this can happen if there's no introduction or very little content.

Additionally, instead of `max-height`, use `max-block-size` for a logical property (`max-width` is accordingly replaced with `max-inline-size`). This ensures we're prepared for other languages that don't read horizontal, but vertical.

Finally, especially here you want to use `dvh` instead `vh` because mobile browsers are weird.

---

discussion in office - not sure if these are actually needed at all, make sure to test on different devices with many variations in amount of content etc. :grimacing:

---

Yeah we don't need it..

I'll drop this change/commit

---

I've made a fix for the underlying issue, as discussed in the office.

In the js a `verticalSpaceUsed` calculation is done for mobile views. This calculates the height of the topbar, which is passed to the css as a variable so that the correct `max-block-size` can be set. The problem is, that this calculation is only done on initial paint. Meaning, if you open the page and *THEN* zoom-in, the `max-block-size` will be set to none (because of the missing css variable)

I've added a window resize listener, that triggers the js calculation and sets/updates the css variable.